core/src/main/java/org/mskcc/cbio/oncokb/bo/impl/AlterationBoImpl.java

```diff
@@ -114,6 +117,13 @@ public Alteration findAlteration(Gene gene, AlterationType alterationType, Strin
             alterations.add(matchedAlt);
         }
 
+        if (addEGFRCTD(alteration)) {
+            Alteration alt = AlterationUtils.findAlteration(GeneUtils.getGeneByHugoSymbol("EGFR"), "EGFR CTD");
```

Comment:
From what I see in the curation platform, "EGFR CTD" is corresponding to the name attribute in Alteration model. I doubt the alteration could be found when the cache is not enabled.